data/404/227/361/404227361.geojson

```diff
@@ -490,6 +496,18 @@
     "src:geom":"quattroshapes",
     "src:population":"eurostat",
     "src:population_date":"2022",
+    "statoids:areakm":87268,
```

Comment:
Double checking these are not floats in the raw Statoids files, right?

Reply:
Correct, Statoids has ints not floats.

Reply:
I strip out any commas from Statoids, convert to float and then convert to an int ... because Python.